src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -388,13 +388,13 @@ $(function () {
         var source_label = get_label_text_for_id(source.id);
 
         var $first_ticket_form = $(".questions-form").first().find("[data-addonidx=0]");
-        var $candidates = $first_ticket_form.find(source.tagName + ":not([type='checkbox'], [type='radio'])");
+        var $candidates = $first_ticket_form.find(source.tagName + ":not([type='checkbox'], [type='radio'], [type='hidden'])");
         var $match = $candidates.filter(function() {
             return (
                 this.id.endsWith(source.id.substring(3))
```

Comment:
```suggestion
                (this.id && source.id && this.id.endsWith(source.id.substring(3)))
```
müssten wir das hier nicht auch checken?

Reply:
Könnte, aber basierend auf den Selektoren ist immer eine `source.id` gegeben, müssten wir also nicht zwingend testen. Und `this.id` ist immer ein String, nur eben ein leerer. Aber man könnte das noch leichter machen, dass wir nur Felder mit id-Attribut auswählen. Also im Selektor so einschränken, dass wir nur Inputs mit nicht-leerem ID-Attribut wählen:

`var $candidates = $first_ticket_form.find(source.tagName + "[id]:not([id=""], [type='checkbox'], [type='radio'], [type='hidden'])");`

Reply:
ah, true, ja dann können wir das lassen